apps/meteor/tests/e2e/omnichannel/omnichannel-chat-history.spec.ts

```diff
@@ -31,9 +31,11 @@ test.describe('Omnichannel chat history', () => {
 		await api.delete('/livechat/users/agent/user1');
 		await api.delete('/livechat/users/manager/user1');
 		await agent.page.close();
+
+		await api.post('/permissions.update', { permissions: [{ _id: 'view-c-room', roles: ['admin', 'owner', 'moderator', 'user'] }] });
```

Comment:
The changeset says `preview-c-room`, it seems one or the other is wrong, isn't it? 🤔